integration-tests/features/environment.py

```diff
@@ -47,6 +47,7 @@ def _skip_test_group(context, test_group):
     return skip_group
 
 def before_all(context):
+    1/0 # Force behave invocation failure to check CI error reporting
```

Comment:
An eariler run was reported as green in GitHub, even though the tests had actually failed. So this is here temporarily to check that `behave` returning a non-zero exit status actually leads to the CI being reported as failing.